packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -561,6 +625,20 @@ const extractTemplateLiteral = (node: t.TemplateLiteral, meta: Metadata): CSSOut
       return acc + quasi.value.raw + suffix;
     }
 
+    /**
+     * Manipulate the node if the CSS property, pseudo classes or pseudo elements are defined
+     * within a template element rather than in a expression.
+     * Eg :hover { color: ${({ isPrimary }) => (isPrimary ? 'green' : 'red')}; }
+     */
+    if (
```

Comment:
We have a function called `extractConditionalExpression` would it make sense to move this logic there? Or does this behaviour only apply to template literals?

Reply:
This extra logic is actually needed to allow  `extractConditionalExpression` evaluate all cases since we currently [expect the property to be defined within the expression](https://github.com/atlassian-labs/compiled/blob/master/packages/babel-plugin/src/utils/css-builders.tsx#L320). I think it should be within `extractTemplateLiteral` since we are manipulating both `expressions` and `quasis`